X-Science/StatusWindow.cs

```diff
@@ -323,7 +323,7 @@ public void UpdateSituation( object sender, NewSituationData e )
 			_logger.Trace( "ScienceThisBiome: " + _filter.TotalCount + " / " + _filter.CompleteCount );
 			if( _filter.TotalCount > 0 )
 			{
-				if( _filter.TotalCount - _filter.CompleteCount > 0 )
+				if( _filter.DisplayScienceInstances.Any(x => CanRunExperiment(x)) )
```

Comment:
well, any linq can be converted to a basic loop more or less easily. but i am not sure if it would save us much in terms of performance. premature optimization is the root of all evil and all that :) but let me try and create a new pull request that gets rid of the linq and then you can decide to adopt either.